ChessCrafterDesktop/src/main/java/com/drewhannay/chesscrafter/panel/PieceCrafterMasterPanel.java

```diff
@@ -41,16 +42,24 @@ private void initComponents() {
         });
 
         JScrollPane scrollPane = new JScrollPane(mPieceList);
+        scrollPane.setFont(new Font(mPieceList.getName(), Font.PLAIN, 26));
         scrollPane.setPreferredSize(new Dimension(190, 500));
 
-        add(scrollPane);
+        setLayout(new GridBagLayout());
+        GridBagConstraints constraints = new GridBagConstraints();
+
+        constraints.fill = GridBagConstraints.BOTH;
+        constraints.weightx = .5;
+        constraints.weighty = 1.0;
+        constraints.insets = new Insets(25,25,25,25);
```

Comment:
`Ctrl+Alt+L` runs the code formatter on Windows, I believe. There should be spaces between the `25, 25, 25, 25`, so i think you need to run the formatter.